### PR TITLE
Turkish language

### PR DESCRIPTION
**Pull request by guneysus**
Reason for this pull request:
- Turkish language support is being added

### CODE REVIEW

**Comment by simonschaufi** [comment]
Thank you. I will merge it, if translation is complete

File: language/tr.php [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+    "recommended" => "önerilenler",
+    "submit" => "Tamam",
+    "cancel" => "İptal",
+    "edit" => "Kaydı Düzenle",
+    "editNote" => "Notu Düzenle",
+    "note" => "Not",
+    "close" => "Kapat",
+    "construction" => "yapım aşamasında...",
+    "about" => "Kimai Hakkında",
+    "hours" => "hours",
```
Comment (simonschaufi) [inline language/tr.php]: missing

File: language/tr.php [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+    "countryCode" => "tr-TR",
+    "preferences" => "Ayarlar",
+    "total" => "Toplam",
+    "monatzeit" => "Ay",
+    "heute" => "Bugün",
+    "gestern" => "Dün",
+    "autoOn" => "Auto On",
+    "autoOff" => "Auto Off",
+    "datum" => "Tarih",
+    "in" => "In",
```
Comment (simonschaufi) [inline language/tr.php]: missing

File: language/tr.php [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+    "starttime" => "Başlangıç Saati",
+    "DateTimeNotInFuture" => "Gün ve saat ileri tarihli olamaz.",
+    "durationlabel" => "Süre",
+    "customer" => "Müşteri",
+    "project" => "Proje",
+    "activity" => "Aktivite",
+    "customers" => "Müşteriler",
+    "projects" => "Projeler",
+    "activities" => "Aktiviteler",
+    "admin" => "Admin",
```
Comment (simonschaufi) [inline language/tr.php]: missing

File: language/tr.php [hunk 4]
```diff
@@ -0,0 +1,537 @@
...
+    "rowlimit" => "max. rows",
+    "breiteZeitspalte" => "width time column",
```
Comment (simonschaufi) [inline language/tr.php]: missing

File: language/tr.php [hunk 5]
```diff
@@ -0,0 +1,537 @@
...
+    "about" => "Kimai Hakkında",
+    "hours" => "hours",
+    "mins" => "minutes",
+    "secs" => "seconds",
+    "skin" => "Tema",
+    "erase" => "Kaydı Sil",
+    "new_customer" => "müşteri ekle",
+    "new_project" => "proje ekle",
+    "new_activity" => "aktivite ekle",
+    "day_summary" => "Time summary",
```
Comment (simonschaufi) [inline language/tr.php]: missing

File: language/tr.php [hunk 6]
```diff
@@ -0,0 +1,537 @@
...
+    "plzChoose" => "seçiniz",
+    "recordAgain" => "Record this recording again",
```
Comment (simonschaufi) [inline language/tr.php]: missing

File: language/tr.php [hunk 7]
```diff
@@ -0,0 +1,537 @@
...
+    'table_date_format' => 'Tablolar için tarih formatı (<a href="http://php.net/manual/en/function.strftime.php" target="_blank">notation for strftime()</a>)',
+    'table_time_format' => 'Tablolar için saat formatı (<a href="http://php.net/manual/en/function.strftime.php" target="_blank">notation for strftime()</a>)',
+    'round_time' => 'Saati şuna yuvarla ',
+    'round_time_minute' => ' dakika.',
+    'allowRoundDown' => 'Aşağı yuvarlamaya müsaade et.',
+    'logged_in_as' => 'olarak giriş yapıldı:',
+    'decimal_separator' => 'ondalık ayracı',
+    'view_filter' => 'filtreyi göster',
+    'unassigned' => 'atanmadı',
+    'all_activities' => '&lt;&lt;all&gt;&gt;',
```
Comment (simonschaufi) [inline language/tr.php]: this doesn't look correct

**Review by simonschaufi — CHANGES_REQUESTED** [review]

File: language/tr.php (hunk 7 above)
Comment (guneysus) [reply language/tr.php]: I translated `view_filter` to meaning `show the filter`. Does not it correct?.
And changed `unassigned` to meaning `Things that are not assigned`. 
`decimal_seperator` and `all_activities` should be correct.

Comment (simonschaufi) [reply language/tr.php]: looks good now.

**Review by simonschaufi — APPROVED** [review]

**Pull request merged**